src/main/webapp/admin_dashboard_script.js

```diff
@@ -1,30 +1,161 @@
-google.charts.load('current', {'packages':['line']});
-google.charts.setOnLoadCallback(drawChart);
+var map;
+var styledMapType;
+var map_key = keys.MAP_KEY;
```

Comment:
The README for this repo says we need `config.MAPS_KEY` in our config.js file. Can we standardize on that or `keys.MAP_KEY` so that we don't have to duplicate it in the config file?